Packages/com.unity.inputsystem/InputSystem/Plugins/OnScreen/OnScreenStick.cs

```diff
@@ -100,32 +111,85 @@ private void Start()
             }
 
             m_StartPos = ((RectTransform)transform).anchoredPosition;
+
+            if (m_Behaviour != Behaviour.ExactPositionWithDynamicOrigin) return;
+            m_PointerDownPos = m_StartPos;
+
+            var dynamicOrigin = new GameObject("DynamicOriginClickable", typeof(Image));
+            dynamicOrigin.transform.SetParent(transform);
+            var image = dynamicOrigin.GetComponent<Image>();
+            image.color = new Color(1, 1, 1, 0);
+            var rectTransform = (RectTransform)dynamicOrigin.transform;
+            rectTransform.sizeDelta = new Vector2(m_DynamicOriginRange * 2, m_DynamicOriginRange * 2);
+            rectTransform.localScale = new Vector3(1, 1, 0);
+            rectTransform.anchoredPosition3D = Vector3.zero;
+
+            image.sprite = SpriteUtilities.CreateCircleSprite(16, new Color32(255, 255, 255, 255));
```

Comment:
Would be useful for users to override this sprite, but because it's only for `Behaviour.ExactPositionWithDynamicOrigin` it's probably ok for now